### PR TITLE
Merge staging

### PR DESCRIPTION
**Pull request by jluk-codi**

### CODE REVIEW

File: hiera/common.yaml [hunk 1]
```diff
@@ -76,6 +76,9 @@ zuul::connections:
 zuul::zookeeper_hosts: "zuulv3.opencontrail.org:3389"
 zuul::statsd_host: "127.0.0.1"
 
+# Configure connection to local statsd collector (collectd plugin)
+zuul::statsd_host: "127.0.0.1"
```
Comment (wurbanski) [inline hiera/common.yaml]: Repeats with https://github.com/Juniper/contrail-infra/pull/129/files#diff-c91fbed12ac130332b2d8eeeff66df62R77

File: modules/opencontrail_ci/manifests/collectd.pp [hunk 2]
```diff
@@ -0,0 +1,113 @@
+class opencontrail_ci::collectd {
+
+  $graphite_host = hiera('opencontrail_ci::graphite_host')
```
Comment (wurbanski) [inline modules/opencontrail_ci/manifests/collectd.pp:3]: Not defined in hiera.

**Review by wurbanski — DISMISSED** [review]

File: modules/opencontrail_ci/manifests/collectd.pp (hunk 2 above)
Comment (jluk-codi) [reply modules/opencontrail_ci/manifests/collectd.pp:3]: Fixed

**Review by kklimonda — APPROVED** [review]
LGTM FWIW

**Review by wurbanski — APPROVED** [review]

**Pull request merged**